python/src/schemas/schema_V2024_05_24.py

```diff
@@ -78,272 +81,515 @@ class CapitalAssetOwnershipType(str, Enum):
+        return self.value
+
 
 class ProjectStatusType(str, Enum):
     NOT_STARTED = "1. Not Started"
     LESS_THAN_FIFTY_PERCENT_COMPLETE = "2. Less than 50 percent complete"
     MORE_THAN_FIFTY_PERCENT_COMPLETE = "3. More than 50 percent complete"
     COMPLETED = "4. Completed"
 
+    def __str__(self):
+        return self.value
+
 
 class YesNoType(str, Enum):
     YES = "Yes"
     NO = "No"
 
+    def __str__(self):
+        return self.value
+
 
 class TechType(str, Enum):
     FIBER = "1. Fiber"
     COAXIAL_CABLE = "2. Coaxial Cable"
     FIXED_WIRELESS = "3. Fixed Wireless"
     OTHER = "4. Other"
 
+    def __str__(self):
+        return self.value
+
 
 class ProjectInvestmentType(str, Enum):
     NEW_CONSTRUCTION = "1. New Construction"
     RENOVATION = "2. Renovation"
 
+    def __str__(self):
+        return self.value
+
 
 class BaseProjectRow(BaseModel):
     model_config = ConfigDict(coerce_numbers_to_str=True, loc_by_alias=False)
 
     row_num: conint(ge=1) = Field(
         default=1,
         serialization_alias="Row Number",
-        json_schema_extra={"column": "NONE"},
+        json_schema_extra={
+            "column": None,
+            "output_column_1A": None,
+            "output_column_1B": None,
+            "output_column_1C": None,
+        },
     )
     Project_Name__c: constr(strip_whitespace=True, min_length=1, max_length=100) = (
         Field(
-            ..., serialization_alias="Project Name", json_schema_extra={"column": "C"}
+            ...,
+            serialization_alias="Project Name",
+            json_schema_extra={
+                "column": "C",
+                "output_column_1A": "B",
+                "output_column_1B": "B",
+                "output_column_1C": "B",
+            },
         )
     )
     Identification_Number__c: constr(
         strip_whitespace=True, min_length=1, max_length=20
     ) = Field(
         ...,
         serialization_alias="Identification Number",
-        json_schema_extra={"column": "D"},
+        json_schema_extra={
+            "column": "D",
+            "output_column_1A": "C",
+            "output_column_1B": "C",
+            "output_column_1C": "C",
+        },
     )
     Subrecipient_UEI__c: constr(strip_whitespace=True, min_length=12, max_length=12) = (
         Field(
             ...,
             serialization_alias="Subrecipient UEI",
-            json_schema_extra={"column": "E"},
+            json_schema_extra={
+                "column": "E",
+                "output_column_1A": "D",
+                "output_column_1B": "D",
+                "output_column_1C": "D",
+            },
         )
     )
     Subrecipient_TIN__c: constr(strip_whitespace=True, min_length=9, max_length=9) = (
         Field(
             ...,
             serialization_alias="Subrecipient TIN",
-            json_schema_extra={"column": "F"},
+            json_schema_extra={
+                "column": "F",
+                "output_column_1A": "E",
```

Comment:
can we name this `treasury_report_col_1A` `treasury_report_col_1B` and `treasury_report_col_1C`?